independent-projects/arc/runtime/src/main/java/io/quarkus/arc/InjectableObserverMethod.java

```diff
@@ -23,13 +26,18 @@ default Set<Annotation> getObservedQualifiers() {
     }
 
     @Override
-    default Reception getReception() {
-        return Reception.ALWAYS;
+    default TransactionPhase getTransactionPhase() {
+        return TransactionPhase.IN_PROGRESS;
     }
 
     @Override
-    default TransactionPhase getTransactionPhase() {
-        return TransactionPhase.IN_PROGRESS;
+    default Bean<?> getDeclaringBean() {
```

Comment:
The `ObserverMethod.getDeclaringBean()` is a CDI 4.0 stuff, right? I'm missing the `@since` javadoc in the API...

Reply:
Actually, yes, I completely forgot it's new.
Is there an issue with it? It's pretty simple to implement for us.

Reply:
No issue really. But it's a pity that the method does not have `@since 4.0` in the javadoc.

Reply:
No doubt about it. I'm just saying that all the new methods should have `@since 4.0` in the CDI API...

Reply:
That is a good point -- I didn't realize you're talking about the CDI API. Let me make a note, I think we should be able to add this retroactively.